apps/admin-gui/src/app/shared/components/dialogs/edit-attribute-definition-dialog/edit-attribute-definition-dialog.component.ts

```diff
@@ -105,7 +115,10 @@ export class EditAttributeDefinitionDialogComponent implements OnInit {
           );
           this.dialogRef.close(true);
         },
-        error: () => (this.loading = false),
+        error: (err) => {
+          this.loading = false;
+          console.error(err);
```

Comment:
Leftover from testing, no?

Reply:
I wanted to leave it here for future better debugging of these several subscriptions and potentially also for E2E tests checking errors in the console. :raised_hands: